enterprise/developerbox/build/README.md

```diff
@@ -1,9 +1,147 @@
 ---
 title: Build Source for Developerbox
 permalink: /documentation/enterprise/developerbox/build/
-redirect_from:
-
 ---
-# Build from Source
 
-Coming soon...
+# Build System Firmware From Source
+
+The System Firmware consists of Trusted Firmware, EDK2 and the
+supporting Developerbox drivers and configuration.
+
+Note: *These instructions will work, without modification, both and x86
+      PC and on your Developerbox.*
+
+
+## Preparation
+
+Firstly, in addition to the "normal" build tools you will also need a
+few specialist tools. On a Debian or Ubuntu operating system try:
+
+~~~
+sudo apt install acpica-tools device-tree-compiler uuid-dev
+~~~
+
+Secondly, create a new working directory and store the absolute path to
+this directory in an environment variable, `WORKSPACE`. It does not
+matter where this directory is created but as an example:
+
+~~~
+export WORKSPACE=$HOME/build/developerbox-firmware
+mkdir -p $WORKSPACE
+~~~
+
+## Cloning the sources
+
+Run the following commands to clone the source code:
+
+~~~
+cd $WORKSPACE
+git clone git://git.linaro.org/leg/noupstream/arm-trusted-firmware.git -b synquacer
+git clone git://git.linaro.org/leg/noupstream/edk2-platforms.git -b developer-box
+git clone git://git.linaro.org/leg/noupstream/edk2-non-osi.git -b developer-box
+git clone git://git.linaro.org/leg/noupstream/edk2.git -b developer-box
+git clone https://github.com/openssl/openssl  -b OpenSSL_1_1_0e
```

Comment:
Since you already install build dependencies earlier, it isn't necessary to build your own openssl library. fwiw, we install libssl-dev package in the CI. We don't build our own.

Reply:
AFAIK this is source for the crypto routines within EDK2... libssl-dev cannot substitute here.

Reply:
do you mean our CI produced firmware aren't working?

Reply:
I have made no statements about the CI builds...

What I know (and have just double checked) is that if the git clone of openssl is removed from the above instructions then edk2 for Developerbox fails to compile even with libssl-dev installed.

Are the CI builds you mentioned for Developerbox or for some other platform. If for another platform then does that platform enable CryptoPkg/OpensslLib.